doc/source/getting_started/docker/linux_container.rst

```diff
@@ -0,0 +1,178 @@
+
+       .. tab-item:: Windows CMD
+
+           .. code-block:: bash
+
+               SET GH_USERNAME=<my-github-username>
+               type GH_TOKEN.txt | docker login ghcr.io -u %GH_USERNAME% --password-stdin
+
+
+#. Pull the Geometry service locally using Docker with a command like this:
+
+   .. code:: bash
+
+      docker pull ghcr.io/ansys/geometry:linux-latest
+
+.. _ref_linux_docker_fromscratch:
+
+Building the Geometry Service Linux container
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Geometry service Docker containers can be easily built by following
+these steps.
+
+Inside the repository's ``docker`` folder, there are two ``Dockerfile`` files:
+
+* ``Dockerfile.linux``: this file builds the Linux-based Docker image.
+* ``Dockerfile.windows``: this file builds the Windows-based Docker image.
+
+Depending on the characteristics of the Docker engine installed in your
+machine, either one or the other has to be built.
+
+In this guide, focus is set on building the ``Dockerfile.linux`` image.
+
+Prerequisites
+~~~~~~~~~~~~~
+
+* Ensure that ``docker`` is installed in your machine.
+  If you do not have ``docker`` available, please refer to
+  :ref:`Running Linux Docker containers <ref_running_linux_containers>`.
+
+* Download the `latest Linux Dockerfile <https://github.com/ansys/pyansys-geometry/blob/main/docker/Dockerfile.linux>`_.
+
+* Download the `latest release artifacts for the Linux
+  Docker container (ZIP file) <https://github.com/ansys/pyansys-geometry/releases/latest/download/linux-binaries.zip>`_.
+
+* Move this ``.zip`` file to the location of the Linux Dockerfile previously downloaded.
+
+Building the Docker images
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to build your image, follow the next instructions:
+
+#. Locate yourself at the folder where the ``.zip`` file and the Dockerfile are placed.
+#. Run the following Docker command:
+
+  .. code:: bash
+
+     docker build -t ghcr.io/ansys/geometry:linux-latest -f Dockerfile.linux .
+
+#. Check that the image has been created successfully. You should see an output similar
+   to this one when running the following command:
+
+  .. code:: bash
+
+     docker images
+
+     >>> REPOSITORY                                               TAG                                IMAGE ID       CREATED          SIZE
+     >>> ghcr.io/ansys/geometry                                   linux-******                       ............   X seconds ago    Y.ZZGB
+     >>> ......                                                   ......                             ............   ..............   ......
+
+
+.. START - Include the common text for launching the service from a Docker Container
+
+.. include:: ./common_docker.rst
+
+.. END - Include the common text for launching the service from a Docker Container
+
+.. button-ref:: index
+    :ref-type: doc
+    :color: primary
+    :shadow:
+    :expand:
+
+    Go back to *Geometry service using Docker*
```

Comment:
Why the use of italic here. The other "Go back to" button didn't have. We could also probably delete "back" and have the button be "Go to ....".

Reply:
Sounds great - will do that